### PR TITLE
[ERE-1917] Cross Form CDE Calculations

### PR DESCRIPTION
**Pull request by ppettitau**
* Remove mnd customisations for patient API extensions as they are no longer required

### CODE REVIEW

File: mnd/mnd/urls.py [hunk 1]
```diff
@@ -23,8 +23,6 @@
     re_path(r'^carer_operations/?$', CarerOperationsView.as_view(), name='carer_operations'),
     re_path(r'^(?P<registry_code>\w+)/carer_registration/?$', MNDRegistrationView.as_view(), name='carer_registration'),
 
-    re_path(r'^api/v1/', include(('mnd.services.rest.urls.api_urls', 'api_urls'), namespace='v1')),
```
Comment (ScriptSmith) [inline mnd/mnd/urls.py:26]: This breaks all api urls because of line 13

Comment (ScriptSmith) [reply mnd/mnd/urls.py:26]: I think just refactor to do urls the same way angelman does

Comment (ppettitau) [reply mnd/mnd/urls.py:26]: Ooops...!
I think I've fixed it now. I tried a couple of the API endpoints and they looked like they were working.

Comment (ppettitau) [reply mnd/mnd/urls.py:26]: p.s. I force pushed the changes.

**Pull request merged**